src/adminserver/systemcfg/systemcfg.go

```diff
@@ -249,13 +251,20 @@ func Init() (err error) {
 	loadAll := false
 	cfgs := map[string]interface{}{}
 
-	if os.Getenv("RESET") == "true" {
-		log.Info("RESET is set, will load all configurations from environment variables")
+	reloadKey := os.Getenv("RELOAD_KEY")
```

Comment:
Maybe add some comment to explain why we need to introduce this RELOAD_KEY?

Additionally we didn't test the new code...

Reply:
Updated